products/resourcemanager/api.yaml

```diff
@@ -93,3 +93,62 @@ objects:
             name: 'id'
             description: Id of the organization
 
+  - !ruby/object:Api::Resource
```

Comment:
Suggestion: To make sure a new `api.yaml` works as expected we use to add to 2 providers at least. Even if they are separate PRs it is worth making sure nothing is inherently provider specific here.